### DISCUSSION TITLE
有没有其他视频网站的插件

### DISCUSSION THREAD

**Discussion by hackerwind** [the1812/Bilibili-Evolved#3584] [提问 / Q&A] [answered]
有没有其他视频网站的插件呀

**Comment by Wkstr** [comment, marked as answer]
在Greasy Fork上应该能找到各站的
推荐个脚本
https://greasyfork.org/zh-CN/scripts/30545-html5%E8%A7%86%E9%A2%91%E6%92%AD%E6%94%BE%E5%B7%A5%E5%85%B7

**Reply by hackerwind** [reply]
好的，谢谢
